src/components/EventForm.js

```diff
@@ -117,17 +120,27 @@ export default function EventForm({ onEventSave, packlists }) {
         <InputLabelColumn1 row={9} htmlFor="EventStartDate">
           Duration:
         </InputLabelColumn1>
-
-        <EventStartDateInput
-          placeholder="yyyy-mm-dd"
-          id="EventStartDate"
-          name="eventStartDate"
-          ref={register({
-            required: true,
-            pattern: /([12]\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))/,
-            validate: (value) => value <= endDateRef.current.value,
-          })}
-        />
+        <EventStartDateDiv>
```

Comment:
Maybe EventStartDateContainer or something similar would be a better naming choice than Div